libdevcrypto/WordList.cpp

```diff
@@ -0,0 +1,5041 @@
+	"fighting",
+	"impressive",
+	"competitor",
+	"ugly",
+	"worried",
+	"portray",
+	"powder",
+	"ghost",
+	"persuade",
+	"moderate",
+	"subsequent",
+	"continued",
+	"cookie",
+	"carrier",
+	"cooking",
+	"frequent",
+	"ban",
+	"swing",
+	"orange",
+	"awful",
+	"admire",
+	"pet",
+	"miracle",
+	"exceed",
+	"rhythm",
+	"widespread",
+	"killing",
+	"lovely",
+	"sin",
+	"charity",
+	"script",
+	"tactic",
+	"identification",
+	"transformation",
+	"everyday",
+	"headline",
+	"crash",
+	"venture",
+	"invasion",
+	"military",
+	"nonetheless",
+	"adequate",
+	"piano",
+	"grocery",
+	"intensity",
+	"exhibit",
+	"high",
+	"blanket",
+	"margin",
+	"principal",
+	"quarterback",
+	"mouse",
+	"rope",
+	"concrete",
+	"prescription",
+	"African-American",
+	"chase",
+	"document",
+	"brick",
+	"recruit",
+	"patch",
+	"consensus",
+	"horror",
+	"recording",
+	"changing",
+	"painter",
+	"colonial",
+	"pie",
+	"sake",
+	"gaze",
+	"courage",
+	"pregnancy",
+	"swear",
+	"defeat",
+	"clue",
+	"reinforce",
+	"win",
+	"confusion",
+	"slice",
+	"occupation",
+	"dear",
+	"coal",
+	"sacred",
+	"criminal",
+	"formula",
+	"cognitive",
+	"collective",
+	"exact",
+	"uncle",
+	"square",
+	"captain",
+	"sigh",
+	"attribute",
+	"dare",
+	"okay",
+	"homeless",
+	"cool",
+	"gallery",
+	"soccer",
+	"defendant",
+	"tunnel",
+	"fitness",
+	"lap",
+	"grave",
+	"toe",
+	"container",
+	"virtue",
+	"abroad",
+	"architect",
+	"dramatically",
+	"makeup",
+	"inquiry",
+	"rose",
+	"surprisingly",
+	"highlight",
+	"decrease",
+	"indication",
+	"rail",
+	"anniversary",
+	"couch",
+	"alliance",
+	"hypothesis",
+	"boyfriend",
+	"compose",
+	"peer",
+	"mess",
+	"rank",
+	"legend",
+	"regulate",
+	"adolescent",
+	"shine",
+	"norm",
+	"upset",
+	"remark",
+	"resign",
+	"reward",
+	"gentle",
+	"related",
+	"organ",
+	"lightly",
+	"concerning",
+	"invent",
+	"laughter",
+	"fit",
+	"northwest",
+	"counseling",
+	"tight",
+	"receiver",
+	"ritual",
+	"insect",
+	"interrupt",
+	"salmon",
+	"favor",
+	"trading",
+	"concern",
+	"magic",
+	"superior",
+	"combat",
+	"stem",
+	"surgeon",
+	"acceptable",
+	"physics",
+	"rape",
+	"counsel",
+	"brush",
+	"jeans",
+	"hunt",
+	"continuous",
+	"log",
+	"echo",
+	"pill",
+	"excited",
+	"sculpture",
+	"compound",
+	"integrate",
+	"flour",
+	"bitter",
+	"bare",
+	"slope",
+	"rent",
+	"presidency",
+	"serving",
+	"subtle",
+	"greatly",
+	"bishop",
+	"drinking",
+	"delay",
+	"cry",
+	"acceptance",
+	"collapse",
+	"shop",
+	"pump",
+	"candy",
+	"evil",
+	"final",
+	"finance",
+	"pleased",
+	"medal",
+	"beg",
+	"sponsor",
+	"ethical",
+	"secondary",
+	"slam",
+	"export",
+	"experimental",
+	"melt",
+	"midnight",
+	"net",
+	"curve",
+	"integrity",
+	"entitle",
+	"evident",
+	"logic",
+	"essence",
+	"park",
+	"exclude",
+	"harsh",
+	"closet",
+	"suburban",
+	"greet",
+	"favor",
+	"interior",
+	"corridor",
+	"murder",
+	"retail",
+	"pitcher",
+	"march",
+	"snake",
+	"pitch",
+	"excuse",
+	"cross",
+	"weakness",
+	"pig",
+	"cold",
+	"classical",
+	"estimated",
+	"T-shirt",
```

Comment:
Since the passphrase is case sensitive it might be a good idea to make everything here lowercase